rust/psibase/src/native.rs

```diff
@@ -180,6 +180,15 @@ pub fn kv_put<'a, K: ToKey, V: Packable<'a>>(db: DbId, key: &K, value: &V) {
     kv_put_bytes(db, &key.to_key(), &value.packed())
 }
 
+/// Set a key-value pair if the key is unique
+///
+/// If key already exists, throws an error
+pub fn kv_insert_unique_bytes<K: ToKey>(db: DbId, key: &K, bytes: &[u8]) {
```

Comment:
Name it `.._put_..` for consistency

Reply:
Not sure I agree on that.  The reason we settled on `put` in the first place is that `put` implies that it overwrites existing, which is precisely what this function doesn't do.

Reply:
I left as `kv_insert_unique` to follow the same pattern as the c++ side: https://github.com/gofractally/psibase/blob/sparkplug0025/rust-elections-tables3/libraries/psibase/common/include/psibase/Table.hpp#L479-L490

I will merge as it is for now to unblock me, but let's keep the discussion here...

Maybe I should move this function out of this native file then and add it to the table abstraction keeping the `insert` name? (Since it's just a sugar around the kv get/put ops?)